### DISCUSSION TITLE
It's unclear how to cancel multiple subscriptions

### DISCUSSION THREAD

**Discussion by mcollie1** [pay-rails/pay#970] [Q&A]
**Scenario:**

Customer is able to subscribe additional users as they're created within a host application (adding new users to an account). The subscription is recurring and a per seat subscription. Thus, the current approach to subscribe users. 

When a customer decides to cancel their plan on the host application, I'd like to execute `subscription.cancel`.

However, that only cancels a single subscription. Is there a way to cancel all subscriptions without iterating over `payment_processor.subscriptions (plural)`?

Conversely, for handling adding users as they are created within a host application, would you suggest incrementing the quantity instead? 

**Note:** I'm using the Stripe processor.

There also appears to be a bug with resuming subscriptions. After syncing, the previous paused state returns

**Comment by excid3** [comment]
Yeah, you're using Stripe incorrectly I'd say. If you're charging per seat, you need to use `quantity` to multiply the line item amount by the number of users. A separate subscription for each would be hard to manage.

See the Stripe docs: https://docs.stripe.com/billing/subscriptions/quantities
